tuxemon/states/techniques/__init__.py

```diff
@@ -126,42 +127,42 @@ def open_choice_menu() -> None:
     def initialize_items(
         self,
     ) -> Generator[MenuItem[Technique], None, None]:
-        """Get all player techniques, remove duplicates, sort
-        and add them to menu."""
-        trainer = local_session.player
-        mon_id = uuid.UUID(trainer.game_variables["open_monster_techs"])
-        monster = trainer.find_monster_by_id(mon_id)
-
+        """Get all player techniques."""
         # load the backpack icon
         self.backpack_center = self.rect.width * 0.16, self.rect.height * 0.45
         self.load_sprite(
-            "gfx/sprites/battle/" + monster.slug + "-front.png",
+            self.mon.front_battle_sprite,
             center=self.backpack_center,
             layer=100,
         )
 
         moveset = []
-        for moves in monster.moves:
-            moveset.insert(moves.tech_id, moves.slug)
+        for moves in self.mon.moves:
+            moveset.append(moves)
+
+        output = sorted(moveset, key=lambda x: x.tech_id)
 
-        output = sorted(moveset)
         for tech in output:
-            obj = Technique(tech)
-            type2 = ""
+            obj = Technique(tech.slug)
```

Comment:
Wasn't tech already a Technique? Why are we recreating it?

Reply:
Well done, I almost missed. Yeah, it's a repetition, I'm going to remove it